client/my-sites/site-settings/form-seo.jsx

```diff
@@ -98,7 +101,18 @@ export const SeoForm = React.createClass( {
 	},
 
 	handleMetaChange( event ) {
-		this.setState( { seoMetaDescription: get( event, 'target.value', '' ) } );
+		const metaContent = event.target.value;
+
+		// Don't allow HTML tags in the input field
+		if ( anyHtmlTag.test( metaContent ) ) {
+			this.setState( { hasHtmlTagError: true } );
+			return;
+		}
+
+		this.setState( {
+			seoMetaDescription: metaContent,
+			hasHtmlTagError: false
+		} );
 	},
```

Comment:
hmmmm…looks like we are splitting what need not be split

```js
handleMetaChange( { target: { value: seoMetaDescription = '' } } ) {
	const hasHtmlTagError = anyHtmlTag.test( seoMetaDescription );

	this.setState( Object.assign( 
		{ hasHtmlTagError },
		! hasHtmlTagError && { seoMetaDescription }
	) );
}
```

Of course, if updating the meta description is okay even if in error, you should shorten even further…

```js
const handleMetaChange = ( { target: { value: seoMetaDescription = '' } } ) => this.setState( {
	seoMetaDescription,
	hasHtmlTagError: anyHtmlTag.test( seoMetaDescription )
} );
```